### ISSUE TITLE
The attention mask is not set and cannot be inferred from input

### ISSUE THREAD

**Issue by ywangwxd** [Vaibhavs10/insanely-fast-whisper#261]
hi all

I saw this warning message when test the CLI,  is this important?

> The attention mask is not set and cannot be inferred from input because pad token is same as eos token. As a consequence, you may observe unexpected behavior. Please pass your input's `attention_mask` to obtain reliable results.
